requirements/prod.txt

```diff
@@ -16,3 +16,4 @@ icalendar
 attrdict
 django-fullurl
 django-apptemplates
+django-dbml
```

Comment:
am i wrong in thinking this is a dev or test requirement rather than prod? it only ever runs in CI or locally, right?

Reply:
Agree that it should be a dev requirment.